src/logger.ts

```diff
@@ -0,0 +1,97 @@
+import { Logger, LoggerModes } from '@overnightjs/logger'
+import morgan from 'morgan'
+import { config } from 'node-config-ts'
+import * as rfs from 'rotating-file-stream'
+
+enum LogLevel {
+  debug,
+  important,
+  warn
+}
+
+const logLevelMapping = {
+  'debug': LogLevel.debug,
+  'important': LogLevel.important,
+  'warn': LogLevel.warn
+}
+
+const LOG_LEVEL = logLevelMapping[config.logger.level] || LogLevel.important
+const LOG_TO_FILES = config.logger.logToFiles || false
+const LOG_PATH = config.logger.filePath || '/usr/local/var/log/cas'
+const METRICS_LOG_PATH = config.logger.metricsFilePath || LOG_PATH + '/metrics'
+
+/**
+ * Handles logging
+ * TODO: Update doc comments
+ */
+export class CASLogger {
+  public readonly logLevel: LogLevel
+  public readonly expressLoggers: any[]
+  private consoleLogger: Logger
+  private includeStackTrace: boolean
+
+  constructor(logLevel: LogLevel = LogLevel.warn) {
+    this.consoleLogger = new Logger(LoggerModes.Console, '', true)
+    this.logLevel = logLevel
+    this.includeStackTrace = this.logLevel == LogLevel.debug ? true : false
+  }
+
+  // Used for stream interface
+  public write(content: string | object): void {
+    this.info(content)
+  }
+
+  public info(content: string | object): void {
+    if (this.logLevel > LogLevel.debug) return
+    this.consoleLogger.info(content, this.includeStackTrace)
+  }
+
+  public imp(content: string | object): void {
+    if (this.logLevel > LogLevel.important) return
```

Comment:
ditto re: greater than or equal to

Reply:
Here if loglevel == important than we __should__ log, not return

Reply:
whoops, missed that 'return' in there.  Aaand this is why I'm not a fan of single-line if-statements :sweat_smile:

Reply:
that's fair